res/database/Default/lang/de.xml

```diff
@@ -128,4 +128,10 @@
 		<programmerole guid="6e48da2f-d468-4542-9510-c3412e8841f7" first_name="" last_name="Wirsch" title="Kommissar" />
 		<programmerole guid="3b128a1f-c2e5-4f76-aa97-0d23d44568d2" first_name="Darlene" last_name="" title="Assistentin" />
 	</programmeroles>
+
```

Comment:
Das Beispiel noch entfernen?

Reply:
Ich hatte die Beispiele extra reingemacht, damit man hier die korrekte Syntax sieht. Ich werde zwar die Dokumentation anpassen, aber sonst gibt es ja kein Beispiel in der Datenbank.